include/ensmallen_bits/bigbatch_sgd/adaptive_stepsize.hpp

```diff
@@ -63,6 +63,7 @@ class AdaptiveStepsize
    * @param iterate Parameters that minimize the function.
    * @param gradient The gradient matrix.
    * @param gradientNorm The gradient norm to be used for the given iteration.
+   * @param sampleVariance Variance in the sample.
```

Comment:
This comment doesn't really make sense to me.  Can you clarify what the parameter actually means?

Reply:
Agreed, we already had the same discussion in the mlpack repository.